src/uu/sort/src/sort.rs

```diff
@@ -583,12 +783,22 @@ fn get_rand_string() -> String {
         .collect::<String>()
 }
 
-fn hash<T: Hash>(t: &T) -> u64 {
-    let mut s: XxHash64 = Default::default();
+fn get_hash<T: Hash>(t: &T) -> u64 {
+    let mut s: FnvHasher = Default::default();
```

Comment:
Would it be possible to have a test covering this line?

Reply:
Yes, but I think only as internal unit tests because unfortunately these are private functions as far as the external tests are concerned.  Please correct me if I'm wrong.

Reply:
sure, this is fine :)